src/mame/mess.flt

```diff
@@ -321,6 +321,7 @@ heathkit/et3400.cpp
 heathkit/h19.cpp
 heathkit/h8.cpp
 heathkit/h89.cpp
+heathkit/tlb.cpp
 hec2hrp/hec2hrp.cpp
 hegenerglaser/academy.cpp
```

Comment:
Don’t add device sources to .flt files – makedep.py scans for dependencies.  It’s only for files with system drivers.